packages/components/src/server-side-render/index.js

```diff
@@ -39,6 +39,7 @@ export class ServerSideRender extends Component {
 	componentDidMount() {
 		this.isStillMounted = true;
 		this.fetch( this.props );
+		this.fetch = debounce( this.fetch, 500 );
```

Comment:
Could we maybe leave a comment explaining why this occurs here vs. in the constructor? It won't necessarily be obvious to the next maintainer.

```suggestion
		// Only debounce once the initial fetch occurs to ensure that the first
		// renders show data as soon as possible.
		this.fetch = debounce( this.fetch, 500 );
```